src/main/java/com/tvtracker/enterprise/dto/UserAccount.java

```diff
@@ -1,13 +1,10 @@
 package com.tvtracker.enterprise.dto;
 
-import lombok.Getter;
-import lombok.Setter;
-
+import lombok.Data;
 import java.sql.Timestamp;
 
-@Getter
-@Setter
-public class UserAccount {
+public @Data
+class UserAccount {
```

Comment:
The @DaTa annotation bundles @tostring, @EqualsAndHashCode, @getter on all fields, @Setter on all non-final fields, and @RequiredArgsConstructor.
https://projectlombok.org/features/Data